src/foam/nanos/dig/DIG.js

```diff
@@ -459,8 +462,7 @@ NOTE: when using the java client, the first call to a newly started instance may
       label: 'Send Request',
       section: "details",
       code: async function() {
-        var url = this.window.location.origin + this.postURL + "&sessionId=" + localStorage.defaultSession;
-        var url = this.postURL + "&sessionId=" + localStorage.defaultSession;
+       var url = this.postURL + "&sessionId=" + this.sessionId || localStorage.defaultSession;
```

Comment:
I think you need need ( )'s around  this.sessionId || localStorage.defaultSession, otherwise you could get:
"&sessionId=undefined"